script/smt-report

```diff
@@ -495,64 +432,22 @@ $LOG = SMT::Utils::openLog($logfile);
 # calculate best report
 my $cfg = SMT::Utils::getSMTConfig();
 
-if ( $cfg->val('NU', 'ApiType', 'NCC') eq 'NCC')
+my $api = $cfg->val('NU', 'ApiType', 'SCC');
```

Comment:
We can think about it. SCC is the only supported ApiType for SMT12. So why not?
But we may want to change it globally?
Or does it make trouble with the migration?